packages/react-storage/src/components/StorageManager/types.ts

```diff
@@ -1,7 +1,5 @@
 import * as React from 'react';
 
-import type { StorageAccessLevel } from '@aws-amplify/core';
-
 import { FileStatus } from '../FileUploader/types';
```

Comment:
Can we just import `StorageAccessLevel` here to dedupe?

Reply:
Not sure what you mean, import from where?

Reply:
From _ ../FileUploader/types_

Reply:
Oh lol yes, updated